ChangeLog

```diff
@@ -1,3 +1,8 @@
+2017-08-04    <adam.johansen@gmail.com>
```

Comment:
Oops, sorry. Missing user-full-name specification in a .emacs file which I'd overlooked. C-x 4 a was indeed what I used, but apparently on a misconfigured system and without paying enough attention on a Friday evening.